### PR TITLE
[RDY] Survivor Zombies Nerf

### PR DESCRIPTION
**Pull request by DeadLeaves**
~~So, like we previously talked about, survivor zombies needed loot nerfs, so that is done. And like @Coolthulhu suggested I added an elite version of the survivor zombie. I tried to balance it to be slightly stronger than a hulk. However it still probably needs some tweaking.~~

I've changed this to only include the well needed nerf of the survivor zombie's loot table. The other might come as a separate PR in the future.

I left in the elite drops if anyone else wants to take a crack at making the elite survivor zombies.

### CODE REVIEW

File: data/json/monsters.json [hunk 1]
```diff
@@ -7366,5 +7370,43 @@
...
+    "special_attacks":[
+        ["SHRIEK", 20],
+        {
+            "type" : "bite",
+            "cooldown" : 5,
+            "min_mul" : 1.0
+        },
+        ["GRAB", 7]
+    ],
+    "description":"This zombie were once a survivor, but now is so much more.  Clad in still functional improvised armor this thing looks like it could quite a beating before going down.  It looks at you with a cunning stare.",
```
Comment (Coolthulhu) [inline data/json/monsters.json]: That "so much more" doesn't fit. It won't be as dangerous as a survivor with a shotgun as long as it stays melee.

File: data/json/monsters.json [hunk 2]
```diff
@@ -7366,5 +7370,43 @@
...
+    "melee_skill":6,
+    "melee_dice":4,
+    "melee_dice_sides":8,
+    "melee_cut":0,
+    "dodge":3,
+    "armor_bash":12,
+    "armor_cut":15,
+    "armor_acid":5,
+    "vision_day":50,
+    "vision_night":3,
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Buff night vision to hunter or even predator levels.

File: data/json/monsters.json [hunk 3]
```diff
@@ -7366,5 +7370,43 @@
...
+    "aggression":100,
+    "morale":100,
+    "speed":110,
+    "melee_skill":6,
+    "melee_dice":4,
+    "melee_dice_sides":8,
+    "melee_cut":0,
+    "dodge":3,
+    "armor_bash":12,
+    "armor_cut":15,
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Needs more armor. Something comparable to best player options, like reinforced heavy survivor armor.

File: data/json/monsters.json [hunk 4]
```diff
@@ -7366,5 +7370,43 @@
...
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":55,
+    "aggression":100,
+    "morale":100,
+    "speed":110,
+    "melee_skill":6,
+    "melee_dice":4,
+    "melee_dice_sides":8,
+    "melee_cut":0,
```
Comment (Coolthulhu) [inline data/json/monsters.json]: For attacks, it needs armor piercing to be balance-able against both good and bad survivors.
Something like
```
"melee_damage" : [
        {"damage_type" : "bashing", "amount" : 20, "armor_multiplier" : 0.5}
    ],
```

File: data/json/monsters.json [hunk 5]
```diff
@@ -7366,5 +7370,43 @@
...
+    "armor_acid":5,
+    "vision_day":50,
+    "vision_night":3,
+    "death_drops": "mon_zombie_survivor_elite_death_drops",
+    "hp":400,
+    "death_function":["NORMAL"],
+    "special_attacks":[
+        ["SHRIEK", 20],
+        {
+            "type" : "bite",
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Probably wears a mask, shouldn't bite

File: data/json/monsters.json [hunk 6]
```diff
@@ -1773,7 +1773,11 @@
         ["GRAB", 7]
     ],
     "description":"Still wearing the tattered remnants of improvised armor and weaponry, it is plain to see that this zombie was once a survivor like you.",
-    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
+    "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "GROUP_BASH", "POISON", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"],
+    "upgrades":{
+        "into": "mon_zombie_survivor_elite",
```
Comment (Coolthulhu) [inline data/json/monsters.json]: Nope
We don't have mechanics of limiting drops to old forms.
Unless it learns how to sew, weld and scavenge while in zombie form, it won't produce the items that elites drop.

Comment (DeadLeaves) [inline data/json/monsters.json]: You mean in the same way a zombie grenadier elite creates c4 hacks? 
Sure I can remove it so it won't evolve into the elite. But then it wont ever spawn afaik.

File: data/json/monsters.json (hunk 1 above)
Comment (DeadLeaves) [inline data/json/monsters.json]: Yeah, like I said, the description is mostly just placeholder, you you (or anyone else for that matter) have a better description. I'll gladly take it.

File: data/json/monsters.json (hunk 4 above)
Comment (DeadLeaves) [inline data/json/monsters.json]: I assume the armor multiplier is armor piercing capabilities? So how does 0.2 sound? Still makes armor work, just not as much.

**Comment by DeadLeaves** [comment]
Sorry about no updates on this, I've been bedridden with the stomach flu. Ill get to this once I'm better

**Comment by pisskop** [comment]
Are survivors going to start dressing up to reflect this change pre-zombification?

Most survivors refuse to even swap up to turnout boots, and prefer light to protective.  Id imagine there would be 'styles', the two most prominent would be the ranged survivor and the melee one.

**Comment by DeadLeaves** [comment]
I've updated the OP and the PR to only include the nerf to regular survivor zombie loot tables.

**Pull request merged**